models/pinterest_ads__ad_adapter.sql

```diff
@@ -18,10 +18,17 @@ with report as (
     select *
     from {{ ref('int_pinterest_ads__most_recent_campaign') }}
 
+), advertiser as (
+    
+    select *
+    from {{ ref('int_pinterest_ads__most_recent_advertiser') }} 
+
 ), joined as (
 
     select 
         report.date_day as campaign_date,
+        advertiser.advertiser_id,
```

Comment:
A couple of q's:
1. Is advertiser_id meant to be similar to account_id in other ad sources?
2. If yes, do you think it'd be beneficial to add advertiser_id & advertiser_name to the other two intermediate tables (ad_group_ad_report & campaign_ad_report)? I notice that we are currently doing that for a couple of other packages e.g. facebook_ads, snapchat.

Reply:
That's a great point. I think it would be beneficial to bring the advertiser information into these other models. I just added these fields to the other models.